src/parquet/file/reader.h

```diff
@@ -34,6 +34,8 @@ struct RowGroupStatistics {
   int64_t num_values;
   int64_t null_count;
   int64_t distinct_count;
+  std::string min;
+  std::string max;
```

Comment:
should these perhaps be `const std::string*`? The questions is whether 

* the `RowGroupStatistics` might have a lifetime independent of the deserialized file metadata or
* copying the min/max is viewed as expensive

In general it's probably not worth worrying too much about large page statistics but curious what you think